src/components/UserTable.js

```diff
@@ -42,7 +42,7 @@ class UserTable extends Component {
         <div>
           <TopPanel />
           <div className="data-table">
-            <table>
+            <table id="userTable">

```

Comment:
We try to avoid using css based on html elements ids.

Reply:
What about using class, like `data-table` in this case?

Reply:
Classes is what we normally do. Check the ABEM naming convention for css.